app/models/institution_tree.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module InstitutionTree
+  def self.build(institution)
+    main = main_campus(institution)
+    all = descendants(main)
+
+    {
+      'main' => {
```

Comment:
I'm curious, why have this 'main' top level key?
Why not just return:
```
      {
        'institution' => main,
        'branches' => build_branches(all),
        'extensions' => build_extensions(all, main.facility_code)
      }
```

Reply:
I agree, but the front-end work is already done and we are going to need to demo this ASAP.